powersimdata/network/usa_tamu/model.py

```diff
@@ -83,16 +80,18 @@ def _drop_interconnect(self):
 def check_interconnect(interconnect):
     """Checks interconnect.
 
-    :param list interconnect: interconnect name(s).
+    :param str/list interconnect: interconnect name(s).
+    :return: (*bool*) -- True if interconnect is valid
```

Comment:
Just noticed we don't really need the return value here - it doesn't return `False` when it fails, so might be simpler to have it either return `None` (by default) or raise an error.

Reply:
We could also return the interconnect as a `list` there are other place in the code where we convert `str` to `list` and we would avoid doing so this way.

Reply:
That feels a bit unexpected given the function name, but if it simplifies things I'm for it

Reply:
I will come up with something more suitable

Reply:
In PostREISE we ended up with `_check_resources_are_in_grid_and_format` to do something similar. It's long, but descriptive.

Reply:
done